app/controllers/admin/themes_controller.rb

```diff
@@ -0,0 +1,56 @@
+class Admin::ThemesController < AdminController
+  before_action :set_theme, only: [:edit, :update, :destroy]
+
+  def index
+    @themes = policy_scope(Theme).order("LOWER(name)")
+  end
+
+  def new
+    authorize Theme
+
+    @theme = Theme.new
+  end
+
+  def edit; end
```

Comment:
@sameera207 theres no authorize for edit?

Reply:
@craighawkes , thanks, I was more thinking, since edit, update and destroy first go through the `set_theme` method it will error out if there is a unauthorised access